sway-ir/tests/constants/cbr_cmp_fold.ir

```diff
@@ -0,0 +1,23 @@
+// regex: ID=[[:alpha:]0-9]+
+
+script {
+    fn test1() -> u64 {
+        entry:
+        v0 = const bool false
+        v1 = const bool false
+        // not: cmp eq
+        v2 = cmp eq v0 v1
+        // not: cbr
+        // check: br get_0_block0
```

Comment:
You're not using the `ID` regex, but maybe you should for the block label?  I have in other tests, just in case somehow its name changes during the pass.  (And in that case the regex will need `_` added.)